app/releases/page.tsx

```diff
@@ -0,0 +1,14 @@
+import Releases from "./Releases";
+import { Suspense } from "react";
+import LoadingText from "@/components/LoadingText";
+
+export default function Page() {
+  return (
+    <div className="mx-auto max-w-4xl p-10">
+      <h1 className="text-4xl pb-10">Recent Releases</h1>
```

Comment:
Move this to the `layout.tsx` of `/app/releases/layout.tsx`, so that <Suspense> need not be used manually. the fallback can be set in `loading.tsx`.

Ref: 
- https://nextjs.org/docs/app/api-reference/file-conventions/layout
- https://nextjs.org/docs/app/api-reference/file-conventions/loading